.gitignore

```diff
@@ -5,7 +5,6 @@
 
 node_modules/
 packages/*/dist/
-packages/*/lib/
```

Comment:
Why remove? Conventionally `lib` in a JS project contains build artifacts. IMO they should not be kept in Git.

Reply:
Furthermore, suggest adding `*.bs.js` here to not duplicate it many times in local `.gitignore`’s

Reply:
Yeah, I agree. That was a silly thing to do. Moved it back to "global" `.gitignore`